package.json

```diff
@@ -37,21 +37,26 @@
 		"babel-core": "^6.26.3",
 		"babel-eslint": "^8.2.2",
 		"babel-loader": "^7.1.2",
+		"babel-plugin-transform-async-generator-functions": "^6.24.1",
 		"babel-plugin-transform-react-jsx": "^6.24.1",
 		"babel-polyfill": "^6.26.0",
 		"clean-webpack-plugin": "^0.1.18",
 		"codecov": "^3.0.1",
 		"concurrently": "^3.5.1",
+		"core-js": "^2.5.5",
 		"cross-env": "^5.1.4",
 		"css-loader": "^0.28.11",
+		"deep-freeze": "0.0.1",
 		"eslint": "^4.19.1",
 		"eslint-config-wordpress": "^2.0.0",
 		"eslint-plugin-jest": "^21.15.1",
 		"eslint-plugin-jsx-a11y": "^6.0.3",
 		"eslint-plugin-react": "^7.7.0",
 		"eslint-plugin-wordpress": "git://github.com/WordPress-Coding-Standards/eslint-plugin-wordpress.git#1774343f6226052a46b081e01db3fca8793cc9f1",
 		"extract-text-webpack-plugin": "^3.0.2",
+		"gutenberg": "Wordpress/gutenberg#master",
 		"pegjs": "^0.10.0",
+		"pegjs-loader": "^0.5.4",
```

Comment:
This is all related to tests implemented for gutenberg type testing.  I anticipate that we'll only be doing this type of testing mostly within the gutenberg branch and thus can just get merged with the shortcode to block conversion but if it turns out this is needed in any future work on master I'll have to cherrypick changes to master and work out the conflicts when merging upstream.